test/Microsoft.OData.ModelBuilder.Tests/Containers/EntitySetConfigurationTest.cs

```diff
@@ -38,7 +37,7 @@ public void CtorThatTakesClrType_Throws_ArgumentNull_For_EntityType()
         [Fact]
         public void CtorThatTakesClrType_Throws_ArgumentNull_For_Name()
         {
-#if NETCOREAPP3_1
+#if NETCOREAPP3_1 || NET5_0
```

Comment:
>NET5_0 [](start = 21, length = 6)

do our model builder test project target on .net5?

Reply:
No. But in my testing I did run a full test against .net5 so I added this.
I consider it "resolving a future bug".